ui/apps/pixano/src/components/layout/DatasetHeader.svelte

```diff
@@ -85,12 +81,24 @@
     return event.key;
   };
 
-  const handleConfirmClick = async () => {
+  const handleSaveandContinue = async () => {
```

Comment:
there is a typo here: `handleSaveAndContinue`

Reply:
Thanks, fixed!